docs/source/installation.rst

```diff
@@ -71,6 +78,9 @@
     {{ app_name }} Python package
     *****************************
 
+    We recommend you `create a virtual Python environment <https://docs.python.org/3/tutorial/venv.html>`_ to work with MatFlow to separate versions for different projects and avoid dependency conflicts.
+    Check that you have the correct environment activated by confirming that the name of the environment appears in brackets in your prompt.
+
     Using pip
     ==========================
 
```

Comment:
If the app version necessary here? If you don't give a version number pip will download hte latest, adn we are updating this version number to the latest every time we push, so just `pip install {{ dist_name }} would have the same effect.

Reply:
Yeah I agree, not necessary.